### ISSUE TITLE
FAB animation

### ISSUE THREAD

**Issue by naXa777** [naXa777/SoundRecorder#28]
Material Design improvement: Add show/hide animation for Floating Action Button

**Comment by elishaolade** [comment]
Hi, I understand this issue is nearly 8 months old, but I want to know if I can still contribute to the show/hide animation for FAB.

**Comment by naXa777** [comment]
@elishaolade yes, you can. and don't worry about failing Travis CI pipelines - it's broken, but the code is working.

**Comment by elishaolade** [comment]
@naXa777 should the button hide only when scolling the list?

**Comment by JuanLH** [comment]
Hello @naXa777, I want to know if the issue still really open, if yes can you give me some tips for handle the problem

**Comment by naXa777** [comment]
Hey @JuanLH,
it's still open.

**Comment by naXa777** [comment]
Expected behavior:

> The show animation grows the widget and fades it in, while the hide animation shrinks the widget and fades it out.

I'm not sure, but it feels like it should be something that we can achieve using standard Material Design libraries.

**Comment by aman0408** [comment]
Can I work on this?

**Comment by naXa777** [comment]
@aman0408 yes, please.

@JuanLH do you have any progress on this issue?

**Comment by aman0408** [comment]
> Expected behavior:
> 
> > The show animation grows the widget and fades it in, while the hide animation shrinks the widget and fades it out.
> 
> I'm not sure, but it feels like it should be something that we can achieve using standard Material Design libraries.

When is this behavior expected?

**Comment by czlucius** [comment]
Can I still work on this?
Thanks.

**Comment by czlucius** [comment]
Which button is it? Is it the record button? It is the only floating action button in the app.
![IMG_20201203_163606](https://user-images.githubusercontent.com/58442255/100984746-13434e80-3586-11eb-9fd7-21db2538b95f.jpg)

**Comment by Emeka-Ugbanu-hub** [comment]
I would love to help with this
